rebar.config

```diff
@@ -1,11 +1,11 @@
 % -*- mode: erlang -*-
 {erl_opts, [debug_info]}.
 
-{port_specs, 
- [{".*-linux", "priv/statfs.so", 
```

Comment:
Yeah, this will indeed try to compile on everything now and will likely fail on many systems.  I could put in configuration for osx and linux specifically. The only reason I didn't initially was that it seems to me that you'll get a build-failure on unsupported systems that will point you in the direction of the code you need to fix.  I might leave it like this for now but put a note in the upstream PR to see what the upstream author prefers.